CHANGELOG.md

```diff
@@ -47,64 +56,100 @@ and removing some features that no longer seem necessary.
+  - BREAKING: `_a_map-multimerge` renamed [`multi-merge()`](utils.html#function--multi-merge)
+  - BREAKING: `_a_str-replace` renamed [`replace()`](utils.html#function--replace)
+  - BREAKING: `_a_interpolate` renamed [`interpolate()`](utils.html#function--interpolate)
+  - BREAKING: `_a_split` renamed [`split()`](utils.html#function--split)
+  - BREAKING: `_a_trim` renamed [`trim()`](utils.html#function--trim)
 
 - [Variables][vars]:
 
-  - NEW: `ident()` function adds `--` and an optional prefix to any string,
+  - NEW: [`ident()`](vars.html#function--ident) function adds `--` and
+    an optional prefix to any string,
     in order to generate a custom property identifier.
-  - NEW: `custom-props()` mixin generates custom properties
+  - NEW: [`custom-props()`](vars.html#mixin--custom-props) mixin generates
+    custom properties
     for every key/value pair in a map.
 
 - [Tokens][token]:
 
   - NEW: A module just for the custom map syntax parser & function registration
-  - BREAKING: The `$functions` map no longer accepts alias references
+  - BREAKING: The [`$functions`](token-register.html#variable--functions) map no
+    longer accepts alias references
     or any other aspects of the token syntax
   - BREAKING: Ratios are no longer first-class adjustments
     (like functions) in the token syntax
-  - BREAKING: `tokens.$handle-missing-keys` defaults to `null`,
+  - BREAKING: [`tokens.$handle-missing-keys`](token-config.html#variable--handle-missing-keys)
+    defaults to `null`,
     and no longer supports the legacy `silent` option.
-  - BREAKING: `get-token()` is renamed `get()`,
+  - BREAKING: `get-token()` is renamed [`get()`](token-api.html#function--get),
     and only accepts bare token names (no `#` prefix),
     including the `outer->inner` nested token syntax.
-  - NEW: The new `compile()` function provides direct access to
-    parse & resolve an arbitrary token value
+  - NEW: The new [`compile()`](token-api.html#function--compile) function
+    provides direct access to parse & resolve an arbitrary token value
     (including aliases with `#` prefix),
     rather than calling a specific token by name.
   - NEW: All the built-in Sass module functions are registered by default,
     as `<module-name>.<function-name>`
-  - NEW: `has-token()` function checks a map
+  - NEW: [`has-token()`](token-inspect.html#function--has-token) function checks a map
     to see if the given token is defined,
     and supports the `outer->inner` nested token syntax.
+  - BREAKING: `_a_plus()` renamed [`plus()`](token-internal.html#function--plus)
+  - BREAKING: `_a_minus()` renamed [`minus()`](token-internal.html#function--minus)
+  - BREAKING: `_a_times()` renamed [`times()`](token-internal.html#function--times)
+  - BREAKING: `_a_divide()` renamed [`divide()`](token-internal.html#function--divide)
+  - BREAKING: `_a_modulo()` renamed [`modulo()`](token-internal.html#function--modulo)
```

Comment:
These can also be documented as new functions, since they were previously private.

Reply:
Changed these to `NEW` functions